client/src/locales/en-US.json

```diff
@@ -346,7 +346,13 @@
             "createFolderFailed": "Failed to create folder `{name}`",
             "listFailed": "Failed to list the content of the folder `{path}`",
             "deleteMultipleAllFailed": "Failed to delete all entries.",
-            "deleteMultipleSomeFailed": "Some entries could not be deleted."
+            "deleteMultipleSomeFailed": "Some entries could not be deleted.",
+            "moveOneFailed": "Failed to move `{name}`",
+            "moveMultipleAllFailed": "Failed to move all entries.",
+            "moveMultipleSomeFailed": "Some entries could not be moved.",
+            "copyOneFailed": "Failed to copy `{name}`",
+            "copyMultipleAllFailed": "Failed to copy all entries.",
+            "copyMultipleSomeFailed": "Some entries could not be copied."
```

Comment:
Can we find out why there is an error? If so, we can tell the user what's going wrong.

Reply:
Ultimately yeah, but it will depends on the actual implementation in the bindings. I can try to invent some of them but it's better to just have a generic message right now and update it when the bindings are available.